vivaldi/hooks/small-vert-pinned-tabs.js

```diff
@@ -0,0 +1,68 @@
+// Small pinned tabs in vertical tab bar
+
+vivaldi.jdhooks.hookModule('yoga-layout', (modInfo, exports) => {
```

Comment:
`hookModule` will be removed in the future, there is a replacement for it - `hookModuleExport`. I hope I'll update wiki soon.

Also I don't like the module's behavior is changed. Is it possible to override the place where it's used - `tabs_TabStrip::createFlexBoxLayout`?

Reply:
OK, I’ll update that. (Not today though.)

Hooking `tabs_TabStrip.createFlexBoxLayout` would be too much work, it creates the layout, *lets it through the yoga layout thing* & *then* returns. If I wanted to change the returned value, I’d have to compute much more (since it includes the *resulting* *absolute* positions of all elements).